audit/format.go

```diff
@@ -406,6 +409,14 @@ func getRemoteAddr(req *logical.Request) string {
 	return ""
 }
 
+// getRemotePort safely gets the remote port avoiding a nil pointer
+func getRemotePort(req *logical.Request) int {
```

Comment:
Same here; we could specify `int32` for example, so that we don't end up with bit inconsistencies in the proto message in edge cases.